specification/iotcentral/data-plane/readme.md

```diff
@@ -20,36 +20,28 @@ openapi-type: data-plane
 tag: package-2022-07-31
 ```
 
-### Tag: package-1.2-preview
```

Comment:
What are all these changes? I only meant that you should set the top-level tag - currently line 20 above - to your latest i.e. "2022-10-31". Why were all these other changes made, or was that just correcting what really exists?

Reply:
According the theory, always put the latest to the top, the order should be from the newest to oldest, isn't your expectation? do you mean change only 2022-10-31-preview one?

Reply:
Order doesn't really matter. My original comment was just about changing the top-level `tag` to be your latest version, as is appropriate and common. Putting latest at the top and ordering descending isn't wrong (it wasn't obvious from the diff - just a wash of green and red with weird blocking on GitHub) but the top-level `tag` still wasn't changed to the latest. I'd keep the diff as clear as possible when changes are irrelevant. Easier to review.

Reply:
I see, some miss understanding here, we only put the latest GA version on the tag, not the latest preview there, will roll back all my changes.